trdg/generators/from_dict.py

```diff
@@ -12,7 +12,7 @@ def __init__(
         count=-1,
         length=1,
         allow_variable=False,
-        fonts=[],
+        fonts="fonts/latin",
```

Comment:
Breaking change here. Please keep the original method signature.

Reply:
Different workflow with fonts.

Reply:
What advantages do you see to giving a directory instead of a list of fonts? Genuinely asking, I would rather not introduce breaking changes, but if you think they are justified it could work for me.

Reply:
For me, I'm working on projects where I need to generate data for OCR recognition. I need to generate images for one OCR model (btw thank you for your CRNN implementation) with lot of specific fonts. 

With this way I can just specify exact folder where are all the required fonts located and not doing it one by one.

Reply:
I understand, but I would prefer if people used `load_fonts` (or their own implementation) on the directory directly and pass that to the generator.

Reply:
Sure, i would try to open another merge request once it will be ready.